developers/weaviate/client-libraries/typescript/typescript-v3.md

```diff
@@ -10,91 +10,93 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 :::note TypeScript client version
-The current TypeScript client version is `v||site.typescript_client_version||`.
+The current Typescript client version is `v||site.typescript_client_version||`.
 :::
 
-## Overview
 
-The TypeScript client can be used for both JavaScript and TypeScript scripts. This page covers the Weaviate TypeScript client; `weaviate-client` on npm. This version is currently in beta. For usage information not specific to the Typescript client, such as code examples, see the relevant pages in the Weaviate documentation. 
 
-## Installation and setup
+The Weaviate TypeScript client supports Javascript and Typescript. The Typescript client v3 is currently in beta. This page covers the Weaviate TypeScript client; `weaviate-client` on the npm registry. 
 
-- Add web cleint initialisation 
-- link to other TS packages and resoruces? like embedded
-- add example queries
-- link to weaviate recipes and example apps
+This client is meant for server development and will not work for web client development. If you need to develop a web client application, please use the `weaviate-ts-client` package or use our [web client](#).
```

Comment:
I think it would be preferable to talk about compatibility than intent here. How about something like:

> This (`weaviate-client`) module is written for use with [Node.js](https://nodejs.org/en), and cannot be used in web applications. For web development, please use the [`weaviate-ts-client`](./index.mdx) module.

Reply:
Sounds good, consider a slight tweak:

This (weaviate-client) module is written for use with [Node.js](https://nodejs.org/en), and cannot be used in web applications. For web development, use the [weaviate-ts-client](https://github.com/weaviate/weaviate-io/pull/index.mdx) module.

Reply:
i prefer this:

> The v3 client currently supports server side development (Node.js hosted). See [v3 packages](#node-support-only) for details

you can use the client for web applications just not on the client side.